### DISCUSSION TITLE
publishing

### DISCUSSION THREAD

**Discussion by klh281** [quarto-dev/quarto-cli#3041] [Q&A]
I developed a quarto website using RStudio. I am now trying to publish it to my new quarto-pub account. I have my quarto-pub account open in my default browser. At the terminal in RStudio, I typed and submitted: quarto publish quarto-pub. I am prompted to Authorize account. I choose Yes, then get the following error:

ERROR: TypeError: error sending request for url (https://quartodev.pub/api/v1/tickets?applicationId=None): error trying to connect: invalid peer certificate contents: invalid peer certificate: CertNotValidForName

Is there another method for providing quarto with my account details?

**Comment by cscheid** [comment]
I just tried the [same thing](https://carlos-scheidegger.quarto.pub/2022-10-26-test/) and can't reproduce your issue. It looks like you're getting a certificate error. Is it possible that your machine has security settings (say, configured by your IT's sysadmins) that change the root certificates that are installed for you? I can imagine something like that causing the issue.

**Reply by klh281** [reply]
Yes, that is it! Thank you.

**Reply by ning3739** [reply]
Did you fix this problem ? any solution?  I also have same problem. Thanks

**Reply by klh281** [reply]
I logged into my computer via my administrator acct and that solved the issue. As cscheid suggested, it must be a permissions issue.

**Reply by ning3739** [reply]
I delate all of my account and try to use token to log in and publishing. It works for me. 

`export QUARTO_PUB_AUTH_TOKEN="your_token"`
